cleaners/vuze.xml

```diff
@@ -52,25 +55,26 @@
   </option>
   <option id="cache">
     <label>Cache</label>
-    <description>Delete the Caches</description>
+    <description>Delete the cache</description>
     <action command="delete" search="file" path="$$profile$$\ipfilter.cache"/>
     <action command="delete" search="walk.all" path="$$profile$$\cache"/>
     <!--<action command="delete" search="walk.all" path="$$profile$$\shares\cache*\"/>-->
-    <!--Seems that not normal caches and shouldn't be deleted!-->
+    <!--Seems that no normal caches and shouldn't be deleted!-->
   </option>
   <option id="logs">
-    <label>Logs and Stats</label>
-    <description>Delete the Logs and Stats</description>
+    <label>Logs</label>
+    <description>Delete the logs</description>
     <action command="delete" search="walk.all" path="$$profile$$\logs\"/>
     <action command="delete" search="glob" path="$$profile$$\plugins\*\*.log"/>
     <action command="delete" search="glob" path="$$profile$$\plugins\*\eventlog.txt"/>
-    <action command="delete" search="walk.all" path="$$profile$$\stats\"/>
-    <action command="delete" search="glob" path="$$profile$$\stats\*"/>
     <action command="delete" search="file" path="$$profile$$\update.log"/>
     <action command="delete" search="file" path="$$profile$$\tracker.log"/>
     <action command="delete" search="glob" path="$$programfiles$$\*.log"/>
     <action command="delete" search="walk.files" path="$$programfiles$$\log\"/>
     <action command="delete" search="glob" path="$$programfiles$$\.install4j\*.log"/>
+    <!-- Delete the stats, too: -->
+    <action command="delete" search="walk.all" path="$$profile$$\stats\"/>
+    <action command="delete" search="glob" path="$$profile$$\stats\*"/>
```

Comment:
This is a similar scenario to other pull reqeusts. Line 77 won't match anything because line 76 deleted the contents already. If you want to delete the folder itself, change line 77 from `glob` to `file` and remove the wildcard from `path`

Reply:
I will try to fix that... (...in all my cleaners!)
...but I have filled an issue about such problems for me here: https://github.com/bleachbit/wishlist/issues/33
Would be nice to get there a feedback! Thx in advance!